src/server.tsx

```diff
@@ -15,10 +15,16 @@ import {
 import { shouldRenderEpic } from './lib/targeting';
 import testData from './components/ContributionsEpic.testData';
 import cors from 'cors';
+import { Validator } from 'jsonschema';
+import * as fs from 'fs';
+import * as path from 'path';
 
 // Pre-cache API response
 fetchDefaultEpicContent();
 
+const schemaPath = path.join(__dirname, 'schemas', 'epicPayload.schema.json');
+const epicPayloadSchema = JSON.parse(fs.readFileSync(schemaPath, 'utf8'));
```

Comment:
Andre pointed this out with the `fetchDefaultEpic` line above, but I think it's also the case here that in production because we're using a lambda the schema loading will actually happen on every request (doing this outside of the request handler doesn't get us anything).

Reply:
Update: we think (along with the default content fetch) that doing this outside of the lambda handler below _does_ mean we cache this value.